File: StrongNumber/StrongNumber.cs
//  strong number: A number is said to be a strong number if the sum of the factorial of the digit is equal to the number itself. Ex. 1, 2, 145
using System;
class Program
{
    static int Factorial(int n)
    {
        int fact = 1;
        while (n > 0)
        {
            fact *= n;
            n -= 1;
        }
        return fact;
    }
    static void Main()
    {
        int n, temp, sum = 0;
        Console.Write("Enter number: ");
        n = int.Parse(Console.ReadLine());
        temp = n;
        while (n > 0)
        {
            sum += Factorial(n % 10);
            n /= 10;
        }
        if (temp == sum)
        {
            Console.WriteLine($"{temp} is a Strong Number");
        }
        else
        {
            Console.WriteLine($"{temp} is not a Strong Number");

        }
    }
}

File: StrongNumber/StrongNumber.c
//  strong number: A number is said to be a strong number if the sum of the factorial of the digit is equal to the number itself. Ex. 1, 2, 145

#include <stdio.h>
int factorial(int n)
{
    int fact = 1;
    while (n > 0)
    {
        fact *= n;
        n -= 1;
    }
    return fact;
}
int main()
{
    int n, sum = 0, temp;
    n = printf("Enter number: ");
    scanf("%d", &n);
    temp = n;
    while (n > 0)
    {
        sum += factorial(n % 10);
        n /= 10;
    }
    if (temp == sum)
    {
        printf("%d is a Strong Number.", temp);
    }
    else
    {
        printf("%d is not a Strong Number.", temp);
    }
    return 0;
}